packages/framework/src/Resources/views/Admin/Inline/Breadcrumb/breadcrumb.html.twig

```diff
@@ -7,7 +7,13 @@
         {% elseif loop.last %}
             <span class="in-breadcrumb__item in-breadcrumb__item--actual">{{ breadcrumbOverrider.lastItemOverridden ? breadcrumbOverrider.lastItemLabel : item.label }}</span>
         {% else %}
+            {% if item.uri is not null %}
+                <a href="{{ item.uri }}">
+            {% endif %}
             <span class="in-breadcrumb__item">{{ item.label }}</span>
+            {% if item.uri is not null %}
+                </a>
+            {% endif %}
```

Comment:
```suggestion
            {% if item.uri is not null %}
                <a href="{{ item.uri }}"><span class="in-breadcrumb__item">{{ item.label }}</span></a>
            {% else %}
                <span class="in-breadcrumb__item">{{ item.label }}</span>
            {% endif %}
```

Reply:
Hello @s3tezsky, wht not. I love latte `n:tag-if` and I'm very sad that twig doesn't have something like that.